kazoo/retry.py

```diff
@@ -129,8 +125,7 @@ def __call__(self, func, *args, **kwargs):
                 if self._attempts == self.max_tries:
                     raise RetryFailedError("Too many retry attempts")
                 self._attempts += 1
-                sleeptime = self._cur_delay + (
-                    random.randint(0, self.max_jitter) / 100.0)
+                sleeptime = random.randint(0, 1+int(self._cur_delay))
```

Comment:
style: missing spaces around the `+`

Reply:
I pushed a style fix along with an updated commit message.